content/en/events/2023/kcsna/_index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Contributor Summit North America 2023"
+type: docs
+weight: 1
+description: >
+  Kubernetes Contributor Summit North America, hosted in Chicago, Illinois.
+
+---
+
+
+### About
+
+The North America Summit will be a lightweight event taking place on November 6th
+in Chicago, Illinois alongside 
+<a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/" rel="noopener noreferrer" target="_blank">KubeCon</a>
+at the
+<a href="https://www.hyatt.com/en-US/hotel/illinois/hyatt-regency-mccormick-place/chimc" rel="noopener noreferrer" target="_blank">Hyatt Regency McCormick Place</a>.
+
+The Summit will be a loosely organized event with a [relaxed schedule](/events/2023/kcsna/schedule),
+ including a Steering AMA, several hours of Unconference Sessions, 
+potential SIG and WG meetings, and a Documentation Sprint.  This will be followed
+by a [Contributor Social](/events/2023/kcsna/social) with games and refreshments.
+
+This page will be updated as more activities are scheduled. If you have
+questions or comments, please reach out to the Summit Staff by emailing
+community@kubernetes.io or asking in the
```

Comment:
All the instances of community@kubernetes.io we'll want to replace with summit-team@kuberrnetes.io (or whatever it is if we want to rename it)

Reply:
Done. Thank you!!